src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -747,8 +717,9 @@ internal abstract class ManagedStrategy(
         // scenario threads are handled separately by the runner itself
         if (isTestThread(currentThreadId)) return
 
+        val eventId = getNextEventId()
```

Comment:
Minor. I think extracting event ID in a separate variable is excessive because it is not used anywhere else unlike thread ID which can be reused.

Reply:
I extracted `val eventId = getNextEventId()` (almost) everywhere for two purposes:

1. Uniformity with `threadId` and between all the injection methods. 

2. As a part of the new contract (documented in one of the comments in the code), `beforeEvent` assumes that each time it is called, previous injection method has advanced the `eventId` for the new event. But some injections (e.g., `afterReadField`) create `TracePoint` conditionally. 

Thus, by moving  `val eventId = getNextEventId()` at the beginning of the methods, I ensure the id allocated for each injection/event, no matter if trace point was created or not.